todo/src/components/TodoForm.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {connect} from 'react-redux'
+import { addNewTodo } from '../actions';
+
+class TodoForm extends React.Component{
+    constructor(){
+        super();
+        this.state={
+            inputText : '',
+        };
+    }
+
+    onChangeHandler = e => {
+        e.preventDefault();
```

Comment:
Don't need to prevent default for changeHandler (inputing text doesn't have problematic default behavior)